Code/CoreData/RKManagedObjectStore.h

```diff
@@ -24,6 +24,8 @@
 
 @class RKManagedObjectStore;
 
+extern NSSet *RKSetOfManagedObjectIDsFromManagedObjectContextDidSaveNotification(NSNotification *notification);
```

Comment:
maybe useful, an alternative is to put it into a private extension not exported in the public headers but imported into the test file rather than redeclare it, at least it's always in one place